assets/css/icons.css

```diff
@@ -1,82 +1,117 @@
 @font-face {
-	font-family: 'thronesdb';
-	src:url('../fonts/thronesdb.eot?irjt2b');
-	src:url('../fonts/thronesdb.eot?#iefixirjt2b') format('embedded-opentype'),
-		url('../fonts/thronesdb.ttf?irjt2b') format('truetype'),
-		url('../fonts/thronesdb.woff?irjt2b') format('woff'),
-		url('../fonts/thronesdb.svg?irjt2b#thronesdb') format('svg');
-	font-weight: normal;
-	font-style: normal;
+  font-family: 'thronesdb';
+  src:  url('fonts/thronesdb.eot?pg0qpl');
+  src:  url('fonts/thronesdb.eot?pg0qpl#iefix') format('embedded-opentype'),
+    url('fonts/thronesdb.ttf?pg0qpl') format('truetype'),
+    url('fonts/thronesdb.woff?pg0qpl') format('woff'),
+    url('fonts/thronesdb.svg?pg0qpl#thronesdb') format('svg');
+  font-weight: normal;
+  font-style: normal;
+  font-display: block;
 }
 
-[class^="icon-"], [class*=" icon-"] {
-	font-family: 'thronesdb';
-	speak: none;
-	font-style: normal;
-	font-weight: normal;
-	font-variant: normal;
-	text-transform: none;
-	line-height: 1;
+.icon {
```

Comment:
this change is the right move, however it breaks the iconography in the deck builder, potentially in other places too (to be seen).

![image](https://user-images.githubusercontent.com/1410427/195735449-27e49819-d636-46d6-ac21-63370c882eaa.png)

correcting this will require some work in the Javascript and template files , basically everywhere we have use any of the `icon-*` classes they need to be accompanies by `icon`.

e.g. `class="icon-baratheon"` renders garbage -> this will have to be `class="icon icon-baratheon"` instead. 

![image](https://user-images.githubusercontent.com/1410427/195735625-6b9e5377-db0b-4ac4-85a4-5c5dc994046e.png)

![image](https://user-images.githubusercontent.com/1410427/195735720-268f5b80-9f96-4208-9440-c063250fa7be.png)

Reply:
files containing `icon-*`classes that may have to be adjusted for this change to happen.

stefan@nichtsnutz: ~/dev/projects/thronesdb on master[$]
$ grep -lr 'icon-' assets/js/
assets/js/app.format.js
assets/js/app.deck_charts.js
assets/js/ui.deckedit.js
assets/js/ui.card.js
assets/js/ui.decklist.js
assets/js/app.deck.js
assets/js/ui.decklist_edit.js
assets/js/translations/messages/es.json
assets/js/translations/messages/en.js
assets/js/translations/messages/en.json
assets/js/translations/messages/es.js
assets/js/app.textcomplete.js

stefan@nichtsnutz: ~/dev/projects/thronesdb on master[$]
$ grep -lr 'icon-' templates/
templates/Decklist/decklist.html.twig
templates/macros.html.twig
templates/base.html.twig
templates/Default/rulesreference.en.html.twig
templates/Default/index.html.twig
templates/Default/rulesreference.es.html.twig
templates/Default/recent_reviews.html.twig
templates/Reviews/reviews.html.twig
templates/Restrictions/cardlist.html.twig
templates/Restrictions/podlist.html.twig
templates/Search/card-faction.html.twig
templates/Search/searchform.html.twig
templates/Search/display-spoiler.html.twig
templates/Search/display-list.html.twig
templates/Search/card-props.html.twig
templates/Search/display-card-reviews.html.twig
templates/Search/card-name-with-link.html.twig
templates/Search/card-name.html.twig
templates/Security/login.html.twig